classes/rental.rb

```diff
@@ -8,26 +8,57 @@ class Rental
   def initialize(date, book, person)
     @date = date
     @book = book
+    book.rentals << self
     @person = person
+    person.rentals << self
+  end
+
+  def to_json(*args)
+    {
+      'date' => @date,
+      'book' => {
+        'title' => @book.title,
+        'author' => @book.author
+      },
+      'person' => {
+        'type' => @person.class.name,
+        'id' => @person.id,
+        'age' => @person.age,
+        'name' => @person.name,
+        'parent_permission' => @person.parent_permission
```

Comment:
- [ ] Nice work, however, we have an error when the user tries to exit the app.
The error is raised on line 28 of the `rental.rb` file, where the code attempts to call parent_permission on an instance of the Teacher class. Based on the error message, the issue seems to be related to the serialization of the Teacher object into JSON format. The `to_json` method is called on a Teacher object, but the `parent_permission` method is not recognized, leading to the `NoMethodError`.
To fix this issue, you need to ensure that the Teacher class has a parent_permission method defined.

Screenshot of error
![image](https://github.com/Kel-nana/school_library/assets/81932478/114ee56a-8845-4f90-ab59-fde98d8f8677)